test/e2e/specs/wp-calypso-gutenberg-page-editor-spec.js

```diff
@@ -540,7 +540,7 @@ describe( `[${ host }] Calypso Gutenberg Editor: Pages (${ screenSize })`, funct
 		step( 'Can insert the payment button', async function () {
 			const pageTitle = 'Payment Button Page: ' + dataHelper.randomPhrase();
 			const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
-			const blockId = await gEditorComponent.addBlock( 'Simple Payments' );
```

Comment:
I _think_ these would have to stay the same to not fail, and then would be updated when we flip the switch.